examples/tornjak/README.md

```diff
@@ -35,38 +47,64 @@ kubectl -n spire-server port-forward service/spire-tornjak-backend 10000:10000
 kubectl -n spire-server port-forward service/spire-tornjak-frontend 3000:3000
 ```
 
-You can now access Tornjak at [localhost:3000](http://localhost:3000).
+You can now access Tornjak with your browser at [localhost:3000](http://localhost:3000).
 
 See [values.yaml](./values.yaml) for more details on the chart configurations to achieve this setup.
 
-## Tornjak and Ingress with ingress-nginx
+## Deploy Tornjak with ingress-nginx
 
-Update examples/production/example-your-values.yaml with your information, most importantly, trustDomain.
+Update your-values.yaml with your ingress information, most importantly, trustDomain, and redeploy
+adding the following:
 
 ```shell
-helm upgrade --install --namespace spire-server spire charts/spire \
---values tests/integration/psat/values.yaml \
---values examples/tornjak/values.yaml \
---values examples/tornjak/values-ingress.yaml \
 --set global.spire.ingressControllerType=ingress-nginx \
---render-subchart-notes --debug
+--values examples/tornjak/values-ingress.yaml
 ```
 
-## Tornjak and Ingress on Openshift
+## Deploy Tornjak with Ingress on Openshift
+
+Obtain the OpenShift Apps Subdomain for Ingress and assign it to environment variable:
 
-When deploying on Openshift, follow the deployment setup as described in
-[Openshift README](../openshift/README.md)
+```shell
+export appdomain=$(oc get cm -n openshift-config-managed  console-public -o go-template="{{ .data.consoleURL }}" | sed 's@https://@@; s/^[^.]*\.//')
+echo $appdomain
+```
 
-Then just add Openshift specific configuration to the above command:
+So it can be passed as follow:
 
 ```shell
---values examples/openshift/openshift-values.yaml
+--set global.openshift=true \
+--set global.spire.trustDomain=$appdomain \
+--set spire-server.ca_subject.common_name=$appdomain \
```

Comment:
These values in particular now differ from https://spiffe.io/docs/latest/spire-helm-charts-hardened-about/installation/#production-deployment

If we do need this section, could these be updated to more align with the other doc?

Reply:
Done. As long as the `trustDomain` matches the `adddomain` all works fine.